recipes/pystackreg/LICENSE.txt

```diff
@@ -0,0 +1,43 @@
+/**
+ * C++ Port of the TurboReg ImageJ Plugin
+ * Original code by Philippe Thevenaz (see below)
+ * Porting by Gregor Lichtner
+ */
+
+/*====================================================================
+| Philippe Thevenaz
+| EPFL/STI/IMT/LIB/BM.4.137
+| Station 17
+| CH-1015 Lausanne VD
+| Switzerland
+|
+| phone (CET): +41(21)693.51.61
+| fax: +41(21)693.37.01
+| RFC-822: philippe.thevenaz@epfl.ch
+| X-400: /C=ch/A=400net/P=switch/O=epfl/S=thevenaz/G=philippe/
+| URL: http://bigwww.epfl.ch/
+\===================================================================*/
+
+/*====================================================================
+| This work is based on the following paper:
+|
+| P. Thevenaz, U.E. Ruttimann, M. Unser
+| A Pyramid Approach to Subpixel Registration Based on Intensity
+| IEEE Transactions on Image Processing
+| vol. 7, no. 1, pp. 27-41, January 1998.
+|
+| This paper is available on-line at
+| http://bigwww.epfl.ch/publications/thevenaz9801.html
+|
+| Other relevant on-line publications are available at
+| http://bigwww.epfl.ch/publications/
+\===================================================================*/
+
+/*====================================================================
+| Additional help available at http://bigwww.epfl.ch/thevenaz/turboreg/
+|
+| You'll be free to use this software for research purposes, but you
+| should not redistribute it without our consent. In addition, we expect
```

Comment:
Based on this line, it sounds like we cannot distribute it. Do they consent to placing this on conda-forge? That could mean it is used for things outside of "research purposes". Is that ok with them? Also later lines go on to request a citation or acknowledgement in published results. What if it is used for something that doesn't have results? Or what if it is used in an application that produces results, but whose results cannot be shared publicly?

For me personally I find this sort of license hard to wrap my head around as it is not a standard OSS license. It seems like we could wind up in trouble or are users could. Though I might be overly worried about this.

Perhaps others can comment on this.

cc @conda-forge/core @conda-forge/staged-recipes

Reply:
I also think it forbids conda-forge from distributing this.

@mrakitin Can you contact the authors to ask them to endorse coda-forge? Or better yet, use a standard OSS licence

Reply:
I am the author of this python package, which itself is just a port of the "stackreg" package written in Java by Philippe Thévenaz. I will ask the author of the original package if he endorses conda-forge and get back to you.